README.md

```diff
@@ -2,4 +2,23 @@
 
 Scaffolding for a [Cockpit](http://www.cockpit-project.org) module.
 
-See [docs/build-notes.md](docs/build-notes.md) for more information on how to build and install this package.
+# Building
+
+Run `npm install` to install dependencies and `make` to build the package. It
+builds into the `dist` directory. Link or copy that to a location were
+`cockpit-bridge` looks for packages.
```

Comment:
could mention `$XDG_DATA_DIRS/cockpit/` explicitly? (please verify the value, just off the top of my head). The paragraph below mentions it, but that doesn't "hang together" well for reading.